docs/advanced/export.md

```diff
@@ -0,0 +1,171 @@
+## Export
+Imported entities are READ ONLY and can only be modified by exporters (exported functions that internally modify the wanted value).
+### Supported Patterns
+
+```yaml
+name: Advanced Import
+```
+#### Semantic: Import
+
+##### Examples
+
+###### Named import
+```ts
+// file0
```

Comment:
Unnecessary file name declaration (also wrong format), the correct format should be `//// file0.ts`, however, it is the same as the default, so please remove this line.